src/agent/onefuzz-task/src/managed/cmd.rs

```diff
@@ -96,7 +96,7 @@ pub async fn run(args: &clap::ArgMatches) -> Result<()> {
 
     let min_available_memory_bytes = 1_000_000 * config.common().min_available_memory_mb;
 
-    // If the memory limit is 0, this will resolve immediately with an error.
+    // If the memory limit is 0, this will never return.
     let check_oom = out_of_memory(min_available_memory_bytes);
```

Comment:
let's disable the check altogether if the memory limit is not >0 instead of launching a task that will loop forever